src/it/projects/MSHADE-382_skip_execution/verify.bsh

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import java.io.*;
+
+File originalJarFile = new File( basedir, "target/original-skip-execution-1.0.jar" );
```

Comment:
Groovy is the standard nowadays, which would probably make this a oneliner, something like:
`assert !(new File( basedir, "target/original-skip-execution-1.0.jar" ).exists())`

Reply:
We replaced this BSF script by Groovy meanwhile.